lib-multisrc/iken/src/eu/kanade/tachiyomi/multisrc/iken/Iken.kt

```diff
@@ -114,35 +127,81 @@ abstract class Iken(
         throw UnsupportedOperationException()
 
     override fun chapterListRequest(manga: SManga): Request {
-        val id = manga.url.substringAfterLast("#")
-        val url = "$baseUrl/api/chapters?postId=$id&skip=0&take=1000&order=desc&userid="
-
-        return GET(url, headers)
+        return GET("$baseUrl/series/${manga.url}", headers)
     }
 
     override fun chapterListParse(response: Response): List<SChapter> {
-        val data = response.parseAs<Post<ChapterListResponse>>()
+        val userId = userIdRegex.find(response.body.string())?.groupValues?.get(1) ?: ""
+
+        val id = response.request.url.fragment!!
+        val chapterUrl = "$baseUrl/api/chapters?postId=$id&skip=0&take=1000&order=desc&userid=$userId"
```

Comment:
user id required?

Reply:
For people that logged in through WebView, yes